Paramore.Brighter.Locking.Azure/AzureBlobLockingProvider.cs

```diff
@@ -11,9 +11,7 @@ public class AzureBlobLockingProvider(AzureBlobLockingProviderOptions options) :
     private readonly BlobContainerClient _containerClient = new BlobContainerClient(options.BlobContainerUri, options.TokenCredential);
```

Comment:
Our style is to include the MIT license statement, in a region, in every source code file, but not in tests

Reply:
Added